GUI/Types/Renderer/OctreeDebugRenderer.cs

```diff
@@ -25,6 +25,33 @@ public static void BindDefaultShaderLayout(int vao, int shaderProgram)
         }
     }
 
+    [StructLayout(LayoutKind.Sequential)]
+    public record struct SimpleVertexNormal(Vector3 Position, Vector3 Normal, Color32 Color)
+    {
+        public static readonly int SizeInBytes = Marshal.SizeOf<SimpleVertexNormal>();
+
+        public SimpleVertexNormal(Vector3 Position, Color32 Color) : this(Position, Vector3.Zero, Color) { }
+
+        public static void BindDefaultShaderLayout(int vao, int shaderProgram)
+        {
+            var positionAttributeLocation = GL.GetAttribLocation(shaderProgram, "aVertexPosition");
+            var normalAttributeLocation = GL.GetAttribLocation(shaderProgram, "aVertexNormal");
+            var colorAttributeLocation = GL.GetAttribLocation(shaderProgram, "aVertexColor");
+
+            GL.EnableVertexArrayAttrib(vao, positionAttributeLocation);
+            GL.EnableVertexArrayAttrib(vao, normalAttributeLocation);
+            GL.EnableVertexArrayAttrib(vao, colorAttributeLocation);
+
+            GL.VertexArrayAttribFormat(vao, positionAttributeLocation, 3, VertexAttribType.Float, false, 0);
+            GL.VertexArrayAttribFormat(vao, normalAttributeLocation, 3, VertexAttribType.Float, false, sizeof(float) * 3);
```

Comment:
Make normal come after color, so you can call `SimpleVertex.BindDefaultShaderLayout` for setting up position/color.